src/styles/Quote.css

```diff
@@ -0,0 +1,85 @@
+.quotes-container {
+  position: relative;
+  transform: none;
+  left: 0;
+  max-width: 76em;
+  margin: auto;
+}
+
+@media only screen and (max-width: 720px) {
+    /* For mobile phones: */
+    [class*="quotes-container"] {
```

Comment:
Indentation is off. Why are we using substring matching of the class name instead of a class selector?